env.example

```diff
@@ -14,7 +14,7 @@ RESTART_POLICY=unless-stopped
 ## Please keep in mind this will create a superuser and it's recommended to use a less privileged
 ## user to connect to the database.
 ## A guide on how to change the database user to a nonsuperuser can be found in docs/creation-of-nonsuperuser.md
-POSTGRES_IMAGE_TAG=14-alpine
+POSTGRES_IMAGE_TAG=18-alpine
```

Comment:
With the update to pg 18, the `docker-compose.yml` file needs be modified to accommodate the `POSTGRES_DATA_PATH` change (which is done). Should we add a note here (or elsewhere) to inform and warn users who may be still using previous versions ? Asking @hanzei for advice.

Reply:
Our docs are open source and available at https://github.com/mattermost/docs/. I would highly appreciate if you submit a PR @brandon1024 :tada: